video/out/opengl/video.c

```diff
@@ -3006,13 +3006,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
     if (!has_frame || !mp_rect_equals(&p->dst_rect, &target_rc)) {
         struct m_color c = p->clear_color;
         float color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
Right, many possibilities, and we keep throwing them around.

I think I made a suggestion how to handle "clear as part of a renderpass". You asked whether we support renderpasses with no vertexes etc. - that should work, as long as there's a valid vertex specification and shaders, even if they are not used. Rendering 0 vertexes will just produce no output. So we could do that.

The second suggestion (having ra_vk translate a single clear() call) would be fine too.